test/README.md

```diff
@@ -82,6 +82,45 @@ environment.
    docker is used only for running drenv tests locally. You can use
    podman for building and running containers locally.
 
+1. Install `podman`
+
+   ```
+   sudo dnf install podman
+   ```
+
+   ramenctl depends on podman, hence it needs to be installed.
+
+   See [Installing podman](https://podman.io/docs/installation) for other options.
+
+## Troubleshooting drenv issues
```

Comment:
This section should be at the end of the document since these issues are not common.

Reply:
This information might be missed as scrolling down to end might not happen always while running into an issue. I think more relevant space would be providing the info right after installation. I can move it to the after "Testing Drenv "  section if you really want.

Reply:
Troubleshooting section should be missed, it is not something you want to read normally. You need it only if you have an issue and this is why it is better at the end of the document (or in another document).

You can add a link to the troubleshooting section at the end of the podman section, something like:

```
For solving common issues see [Troubleshooting drenv issues](#troubleshooting-drenv-issues)
```